scenario-compiler/src/main/scala/pl/touk/nussknacker/engine/compiledgraph/CompiledNodesCollector.scala

```diff
@@ -0,0 +1,38 @@
+package pl.touk.nussknacker.engine.compiledgraph
+
+import pl.touk.nussknacker.engine.compiledgraph.node._
+
+object CompiledNodesCollector {
+
+  def collectNodes(node: Node): List[Node] = {
```

Comment:
maybe `collectAllNodes` will be a little bit less confusing?

Reply:
Done